eland/etl.py

```diff
@@ -77,6 +78,9 @@ def pandas_to_eland(
         * False: Include missing values - may cause bulk to fail
     es_type_overrides: dict, default None
         Dict of field_name: es_data_type that overrides default es data types
+    es_enforce_index_schema: bool, default 'True'
```

Comment:
Let's use `es_verify_mapping_compatibility` for a name here.

Reply:
Sounds good, will make those changes.

Tried `es_enforce_index_schema` to see if that was preferred :p

Reply:
Done!